science/nektarpp/Portfile

```diff
@@ -32,30 +32,19 @@ master_sites            ${master_sites}:main \
                         ${homepage}thirdparty/:thirdparty
 
 set main_distfile       ${distfiles}
-distfiles               ${main_distfile}:main \
-                        tetgen-1.5.zip:thirdparty \
-                        triangle-1.6.zip:thirdparty
-
-checksums               nektar-5.2.0.tar.bz2 \
-                        rmd160  3d7fa153c316cd00fd50d9ab5c22f0c79cd89379 \
-                        sha256  42be94624cd049be4a2d62b89033504f1bfd5e8cda005c3705581b1be7ca63b1 \
-                        size    58312042 \
-                        tetgen-1.5.zip \
-                        rmd160  d8785f4ca6b26608ec9423f7574a0e736380370a \
-                        sha256  52207793198746de14abcb30a0aeed617d7348ff37544e7d7e65aaaa76d7fa70 \
-                        size    264128 \
-                        triangle-1.6.zip \
-                        rmd160  c92c011c442157cae5abcf246935d61bdfeede26 \
-                        sha256  7d74d7efbe35dc5a497bd179bdc87b4670740e466f67043d41e0b0546ce76cb5 \
-                        size    130331
+distfiles               ${main_distfile}:main
+
+checksums               nektar-5.3.0.tar.bz2 \
```

Comment:
file name shouldn't be required here anymore

Reply:
See above (I think?)